includes/data-stores/class-wcs-orders-table-subscription-data-store.php

```diff
@@ -603,4 +603,59 @@ public function get_subscription_customer_ids() {
 
 		return $wpdb->get_col( "SELECT DISTINCT customer_id FROM {$table_name} WHERE type = 'shop_subscription'" ); // phpcs:ignore WordPress.DB.PreparedSQL.InterpolatedNotPrepared
 	}
+
+	/**
+	 * Deletes all rows in the postmeta table with the given meta key.
+	 *
+	 * @param string $meta_key The meta key to delete.
+	 */
+	public function delete_all_metadata_by_key( $meta_key ) {
+		global $wpdb;
+		$table_name = self::get_meta_table_name();
+
+		$wpdb->query(
```

Comment:
**Nitpick:** Could we use [`$wpdb->delete()` ](https://developer.wordpress.org/reference/classes/wpdb/delete/)here instead? It probably doesn't make much difference but it seems like an appropriate use case.

Reply:
Yep thats a good idea and it looks a lot cleaner. Fixed in 8f5914b9